docs/quickstart/aws.md

```diff
@@ -0,0 +1,152 @@
+## Requirements 
+
+This quickstart assumes you have access to admin level access to AWS. 
```

Comment:
"access to admin level access to AWS" 🤔

Reply:
i wrote this without figuring out the min perms to get coder working. hence requiring admin access

Reply:
Yeah I mean the word `access` is redundant. I think you're referring to the `AdministratorAccess` policy?